src/__tests__/article_Tests/__snapshots__/createArticle.test.js.snap

```diff
@@ -0,0 +1,7 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`should render initial layout 1`] = `
+Array [
+  <createArticle />,
+]
+`;
```

Comment:
Snapshots shouldn't be pushed online. Add it to your gitignore

Reply:
This file has been added to the gitignore